packages/edit-site/src/hooks/use-theme-style-variations/use-theme-style-variations-by-property.js

```diff
@@ -71,27 +69,26 @@ export function useCurrentMergeThemeStyleVariationsWithUserConfig( {
 		};
 	}, [] );
 	const { user: baseVariation } = useContext( GlobalStylesContext );
+	const clondedBaseVariation = cloneDeep( baseVariation );
```

Comment:
Typo.

```suggestion
	const clonedBaseVariation = cloneDeep( baseVariation );
```

Reply:
Actually, I don't think there's any reason to do a deep merge here. `mergeBaseAndUserConfigs` uses `deepmerge` which doesn't mutate the original object.

Reply:
Since I fixed a bug in [ba440bd](https://github.com/WordPress/gutenberg/pull/62469/commits/ba440bde6baac3e35960239be7f4522c2e3921fc), I think we do need the clone again..